pom.xml

```diff
@@ -75,7 +79,7 @@
 
     <properties>
         <checkstyle.skip>false</checkstyle.skip>
-        <java.min.version>1.7</java.min.version>
+        <java.min.version>1.8</java.min.version>
```

Comment:
Background makes sense.  The only other user I could find on maven central was java-triton, and java7 is EOL circa 2015, so I'm going to roll forward in this case.